bluesky/tests/test_callbacks.py

```diff
@@ -57,69 +57,25 @@ def cb():
 
 
 def test_subs_input():
-    called = defaultdict(lambda: False)
     def cb1(name, doc):
-        called['cb1'] = True
+        pass
+
     def cb2(name, doc):
-        called['cb2'] = True
+        pass
+
     def cb3(name, doc):
-        called['cb3'] = True
```

Comment:
These tests play badly with OSX for some reason. @tacaswell suggested a better way to test this aspect anyway.